packages/connect-cloudflare/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint --max-warnings 0 ."
   },
   "dependencies": {
-    "@bufbuild/protobuf": "^2.0.0",
+    "@bufbuild/protobuf": "^2.0.1",
```

Comment:
Should this be `2.1.0`?

Reply:
Yes 🙈 Amazing that it still works, right?

Fixed in https://github.com/connectrpc/connect-es/pull/1239/commits/8ab2e5a79184b81a3d220d8e29d0433b0482e847